src/SilverpopConnector/SilverpopXmlConnector.php

```diff
@@ -996,4 +1072,31 @@ protected function post($params, $pathExtension='/XMLAPI', $urlParams='') {
 		curl_close($ch);
 		return $this->checkResponse($result);
 	}
+
+	/**
+	 * Download a file from the sftp server.
+	 *
+	 * @param string $fileName
+	 * @param string $destination
+	 *   Full path of where to save it to.
+	 *
+	 * Sample code:
+	 *
+	 * $status = $this->silverPop->getJobStatus($this->getJobStatus();
+	 *   if ($status === 'COMPLETE')) {
+	 *     $file = $result->downloadFile();
+	 *   }
+	 *
+	 * @return bool
+	 */
+	public function downloadFile($fileName, $destination) {
```

Comment:
This public method should be sorted alphabetically with the other public methods in the top area of the file.